documentation/users-guide/ALittleC++.md

```diff
@@ -2,13 +2,13 @@
 \index{C++}
 
 This chapter introduces you to some useful insights into C++, to allow
-you to use of the most advanced features in ROOT. It is in no case a
+you to use some of the most advanced features in ROOT. It is in no case a
 full course in C++.
 
 ## Classes, Methods and Constructors
 
 
-C++ extends C with the notion of class. If you're used to structures
+C++ extends C with the notion of a class. If you're used to structures
```

Comment:
```suggestion
C++ extends C with the notion of class. If you're used to structures
```

Reply:
Hi!

Thanks for the review. I'm a Google Season of Docs 2019 applicant. I am interested in helping ROOT with the tasks listed on HSF's [website](https://hepsoftwarefoundation.org/gsdocs/2019/proposal_ROOT.html). I am currently in the process of reading and critiquing the existing ROOT docs.

Would love to have your guidance!